bin/cron.js

```diff
@@ -157,7 +163,14 @@ cron.schedule("*/15 * * * *", async () => {
                   if (submission.score >= 90) {
                     score += 100;
                     completed.apply += 1;
-                    if (subj && userProgress.modules[moduleID].practice) {
+                    if (
+                      (subj === "chem" ||
+                        subj === "econ" ||
+                        subj === "physics" ||
+                        subj === "engineering" ||
+                        subj === "bio") &&
+                      userProgress.modules[moduleID].practice
```

Comment:
Why are the application subject quizzes dependent on if the practice has been completed? I kinda see why as the applications should always be taken after the practice ones, but I'm not sure about having it be a requirement in the code. Let me know what you think

Reply:
The badge req has the condition that the ALEKS(practice) is complete.

Reply:
I think I might have made a mistake with the other pr. The sole purpose of this function here should be to count the number of assignments types completed, separate from any badge requirements. The order the submissions come in isn't ordered in any way, so with the way the code is right now, an application quiz that come first won't get counted if the practice one comes after it. They won't get awarded the badge then when they should have. The logic for if they get awarded a badge should be in updateBadgeProgress in canvas.js.